ts/sagas/wallet.ts

```diff
@@ -859,6 +861,12 @@ export function* watchWalletSaga(
 
     // watch for add co-badge to Wallet workflow
     yield takeLatest(walletAddCoBadgeStart, addCoBadgeToWalletAndActivateBpd);
+
+    // watch for add privative to Wallet workflow
```

Comment:
```suggestion
    if(privativeEnabled){
      // watch for add privative to Wallet workflow
      yield takeLatest(
        walletAddPrivativeStart,
        addPrivativeToWalletAndActivateBpd
      );
    }
```

Reply:
Thanks. Solved in [f298052](https://github.com/pagopa/io-app/pull/2869/commits/f298052296693a9b54a39ce3590e93aeb2c2d1df)